### ISSUE TITLE
Helita import statements in Jupyter notebooks

### ISSUE THREAD

**Issue by retifrav** [ITA-Solar/rh#29]
In [sample Jupyter notebooks](https://github.com/ITA-Solar/rh/tree/master/doc/notebooks) the following import statements for Helita are failing:

```
from helita.sim import rh15d, rh15d_vis
```

because it should be this instead:

```
from helita.sim import rh15d
from helita.vis import rh15d_vis
```

The `rh15d_vis.py` was moved there [a year ago](https://github.com/ITA-Solar/helita/commit/e8a5f38586204c15e695a180ac34fd3cc96f7afa#diff-982c64c5ed6104da9467fa7acac30683b57ea686d5480b16eaa1743f13e00835).

I can create a pull-request with fixed imports, if you want.

**Comment by souvikita** [comment]
Sure go ahead!

**Comment by tiagopereira** [comment]
Fixed in 0e84c93fb5cebf62029b36d4ee5dbafdfe9ad, together with an update on the matplotlib backend.
